docs/how-to-guides/adding-a-new-offline-store.md

```diff
@@ -215,7 +219,7 @@ class CustomFileDataSource(FileSource):
 
 After implementing these classes, the custom offline store can be used by referencing it in a feature repo's `feature_store.yaml` file, specifically in the `offline_store` field. The value specified should be the fully qualified class name of the OfflineStore.&#x20;
 
-As long as your OfflineStore class is available in your Python environment, it will be imported by Feast dynamically at runtime.
+As long as your OfflineStore class is available in your Python environment, it will be imported by Feast dynamically at runtime. It is crucial to specify the type as the package that Feast can import.
```

Comment:
Might be more clear if you left this as a comment in the example `feature_store.yaml`